### PR TITLE
Update fluiddem interface

### PR DESCRIPTION
**Pull request by jgonzalezusua**
Updating FluidDEM interface to be able to select _two-way_ problem parameters and fixing bugs in master branch.

### CODE REVIEW

**Comment by GuillermoCasas** [comment]
I would approve but I prefer @maceligueta to please take a quick look at the new features added to the DEM interface. First, search tolerance is added (I believe), second, the phantom walls are added. Both things are important for the regular DEM interface.

File: kratos.gid/apps/DEM/write/writeProjectParameters.tcl [hunk 1]
```diff
@@ -69,7 +69,7 @@ proc DEM::write::getParametersDict { } {
 
     dict set strategy_parameters_dict "RemoveBallsInitiallyTouchingWalls"   [write::getValue AdvOptions RemoveParticlesInWalls]
     dict set strategy_parameters_dict "strategy"                            $dem_strategy
-    dict set project_parameters_dict "solver_settings"                      $strategy_parameters_dict
+    dict set project_parameters_dict  "solver_settings"                      $strategy_parameters_dict
```
Comment (GuillermoCasas) [inline kratos.gid/apps/DEM/write/writeProjectParameters.tcl:72]: extra space

File: kratos.gid/apps/FluidDEM/xml/XmlController.tcl [hunk 2]
```diff
@@ -78,7 +80,7 @@ proc FluidDEM::xml::CustomTree { args } {
 
     # set result_node [$root selectNodes "[spdAux::getRoute FLResults]/container\[@n = 'GiDOptions'\]"]
 	# if { $result_node ne "" } {$result_node delete}
-    spdAux::SetValueOnTreeItem state disabled FLSolStrat
+    spdAux::SetValueOnTreeItem state enabled FLSolStrat
```
Comment (maceligueta) [inline kratos.gid/apps/FluidDEM/xml/XmlController.tcl:83]: If we were disabling this item maybe it's not necessary to enable it explicitly and this line could be removed.

File: kratos.gid/apps/FluidDEM/xml/Main.spd [hunk 3]
```diff
@@ -5,6 +5,7 @@
     <include n="AnalysisType"        active="1" path="apps/Fluid/xml/AnalysisType.spd"/>
     <include n="FluidParts"          active="1" path="apps/Fluid/xml/Parts.spd"/>
     <include n="InitialConditions"   active="1" path="apps/Fluid/xml/InitialConditions.spd"/>
+    <include n="ManufacturedConditions" active="1" path="apps/FluidDEM/xml/ManufacturedConditions.spd"/>
```
Comment (maceligueta) [inline kratos.gid/apps/FluidDEM/xml/Main.spd:8]: Are you sure this is of general interest? I mean, if it's for testing, it should be in the tests, but not in the GUI. No normal user should be interested in using ManufacturedConditions.

Comment (jgonzalezusua) [reply kratos.gid/apps/FluidDEM/xml/Main.spd:8]: I think could be useful. If you know the exact solution of a variable you can impose it by using that condition. In the tree is set as "Exact Conditions"

**Comment by jgonzalezusua** [comment]
At the moment, the ManufacturedConditions are an internal feature we are using and, if it is not of general interest, we can remove it in the future. @maceligueta, can you approve the PR?

**Comment by maceligueta** [comment]
I am approving, but it would be better if this feature is only available to developers (when that option is chosen) as we have spoken in person.

**Review by maceligueta — APPROVED** [review]
Approving to speed up the process but switch to developer mode pending if it is possible

**Review by maceligueta — APPROVED** [review]

**Pull request merged**